Documentation/compatibility/Keytips behavior improved in WPF.md

```diff
@@ -0,0 +1,30 @@
+## [Keytips behavior improved in WPF]
+
+### Scope
+[Edge]
+
+### Version Introduced
+[4.7.2]
+
+### Source Analyzer Status
+[NotPlanned|]
+
+### Change Description
+[Keytips behavior has been modified to bring parity with behavior on Word, Explorer. By checking whether keytip state is enabled or not in the case of a SystemKey (Alt or F11 in particular here) being pressed, we handle keytip keys appropriately. Keytips now dismiss a menu even when opened by mouse]
```

Comment:
"and" rather than comma between Word and Explorer
Word --> Microsoft Word
What is Explorer? Windows Explorer? 
It would be helpful if these are links rather than plain text:
- SystemKey: \<xref:System.Windows.Input.KeyEventArgs.SystemKey%2A>
- Alt: \[Alt\](xref:System.Windows.Input.Key)
- F11: \<xref:System.Windows.Input.Key.F11>
(Alt or F11 in particular here) --> (in particular, Alt or F11)
we handle --> WPF handles
when opened --> when it is opened
period after "mouse"